src/components/article/ViewArticle.jsx

```diff
@@ -37,77 +44,92 @@ class ViewArticle extends React.Component {
-          />
-
-          <div className="top-details">
-            <h1 className="title">{title}</h1>
-            <p className="article-date-author">
-              {date}
-            &nbsp;by &nbsp;
-              {username}
-            </p>
-
-            <div className="read-time">
-              <i className="little material-icons icon">schedule</i>
-              <p>
-                &nbsp;
-                {readtime}
-                &nbsp;
-                read
-              </p>
+          {loading ? (
+            <div className={loading === '' ? 'center display-none' : 'center display-block'}>
+              {loading && <PreLoader />}
             </div>
-
-            <div className="article-rate" />
-          </div>
-
-          <div className="article-main">
-            <p>{body}</p>
-          </div>
-
-          <div className="tag-container">
-            <p>
-              <b>
-                TAGS:&nbsp;
-              </b>
-              {tags ? tags.map(tag => `${tag}, `) : ''}
-            </p>
-          </div>
-
-          <div className="icon-container">
-            <Like
-              handleClick={this.handleFavoriteClick}
-              likeButtonStyle={this.likeButtonStyle}
-              article={this.props}
-              isLike={isLike}
-            />
-          </div>
-
+          ) : (
+            <Fragment>
+              <DisplayMessage message={message} />
+              <div className="top-details">
+                <h1 className="title">{title}</h1>
+                <p className="article-date-author">
+                  {date}
+                  &nbsp;by &nbsp;
+                  {username}
+                </p>
+
+                <div className="read-time">
+                  <i className="little material-icons icon">schedule</i>
+                  <p>
+                    &nbsp;
+                    {readtime}
+                    &nbsp; read
+                  </p>
+                </div>
+                <div className="article-rate" />
+              </div>
+
+              <div className="article-main">
+                <p>{body}</p>
+              </div>
+
+              <div className="tag-container">
+                <p>
+                  <b>TAGS:&nbsp;</b>
+                  {tags ? tags.map(tag => `${tag}, `) : ''}
+                </p>
+              </div>
+
+              <div className="icon-container">
+                <Like
+                  handleClick={this.handleFavoriteClick}
+                  likeButtonStyle={this.likeButtonStyle}
+                  article={this.props}
+                  isLike={isLike}
+                />
+              </div>
+
+              <hr />
+              <div className="article-comments">
+                {isLoggedIn ? (
+                  <CommentForm articleSlug={slug} />
+                ) : (
+                  <p>
+                    Please
+                    {' '}
+                    <Link to="/">log in</Link>
```

Comment:
hi @codeshifu you're saying this should link back to the login page, but the link goes to the index page. It should link to `/login`.

Reply:
@oluwajuwon Thanks, mate.